### PR TITLE
Tag: fix stop propagation problem

### LINKED ISSUE #15149: [Bug Report] remove stopPropagation when click Tag
### Element UI version
2.7.2

### OS/Browsers version
MacOs 10.14.3, Chrome 73

### Vue version
2.6.10

### Reproduction Link
https://jsfiddle.net/75jp80rf/1/

### Steps to reproduce
1. example code in link
2. click one Tag option ( the first option, for example )
3. Select doesn't change its value

### What is Expected?
Select change its value

### What is actually happening?
Select doesn't change its value
The reason is:
In Tag component, it stops propagation when click.
Obviously, developers can use '.stop' modifier if it is necessary.

### PR DESCRIPTION
**Pull request by infjer**
Please make sure these boxes are checked before submitting your PR, thank you!

* [ ] Make sure you follow Element's contributing guide ([中文](https://github.com/ElemeFE/element/blob/master/.github/CONTRIBUTING.zh-CN.md) | [English](https://github.com/ElemeFE/element/blob/master/.github/CONTRIBUTING.en-US.md) | [Español](https://github.com/ElemeFE/element/blob/master/.github/CONTRIBUTING.es.md) | [Français](https://github.com/ElemeFE/element/blob/master/.github/CONTRIBUTING.fr-FR.md)).
* [ ] Make sure you are merging your commits to `dev` branch.
* [ ] Add some descriptions and refer relative issues for you PR.

- Closes #15149

### CODE REVIEW

File: packages/tag/src/tag.vue [hunk 1]
```diff
@@ -12,11 +12,9 @@
     },
     methods: {
       handleClose(event) {
-        event.stopPropagation();
```
Comment (ziyoung) [inline packages/tag/src/tag.vue]: 这里应该没有必要去掉了。

**Comment by luckyCao** [comment]
@infjer 看一下 review 的评论

File: packages/tag/src/tag.vue (hunk 1 above)
Comment (infjer) [reply packages/tag/src/tag.vue]: 这里可以保留。🙂

Comment (luckyCao) [reply packages/tag/src/tag.vue]: @infjer  修改一下吧

**Pull request merged**